src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/data/Prisoners.kt

```diff
@@ -67,4 +68,5 @@ data class Prisoner(
   val immediateNeedsSubmitted: Boolean,
   val preReleaseSubmitted: Boolean,
   val isInWatchlist: Boolean,
+  val profile: ProfileTagList,
```

Comment:
I don't think we will need this in the search results, just on the get a single prisoner

Reply:
yes this Prisoner data  is for singe prisoner. though the data class in Prisoners.kt, it is used for single prisoner only. The search for list prisoners using PrisonersSearch data class

Reply:
Oh it's just the way github had collapsed the diff, made it look like it was part of Prisoners class. Ignore :)